Cargo.Bazel.json.lock

```diff
@@ -33370,14 +33370,14 @@
       ],
       "license_file": "LICENSE"
     },
-    "ic-identity-hsm 0.39.2": {
+    "ic-identity-hsm 0.39.3": {
```

Comment:
Why are these versions of unrelated libraries changing? Is it because for ones whithout any explicit version an update of these lockfiles triggers going to the newest version?